README.md

```diff
@@ -134,8 +135,9 @@ sudo apt install libnotify-dev
 ```text
 sudo apt-get install libcurl4-openssl-dev
 sudo apt-get install libsqlite3-dev
-wget https://github.com/ldc-developers/ldc/releases/download/v1.11.0/ldc2-1.11.0-linux-aarch64.tar.xz
-tar -xvf ldc2-1.11.0-linux-aarch64.tar.xz
+sudo apt-get install libxml2
+wget https://github.com/ldc-developers/ldc/releases/download/v1.13.0/ldc2-1.13.0-linux-aarch64.tar.xz
+tar -xvf ldc2-1.13.0-linux-aarch64.tar.xz
```

Comment:
Shouldn't that be 1.14.0 to be consistent with the travis test file? Ir do you want the same version for all in the README file?

Reply:
PR #413 updates things to 1.14.0, so this updates everything to "1.13.0" + the travis ci process to use the 'right versions'
I will be applying this PR, then #413 in that order